docs/draft/webview.md

```diff
@@ -0,0 +1,119 @@
+        * [`isConnected`](#isConnected)
+        * [`scanQRCode`](#scanQRCode)
+        * [`send`](#send)
+        * [`sendAsync`](#sendAsync)
+        * [`sendSync`](#sendSync)
+5. [Compatibility](#compatibility)
+6. [Changelog](#changelog)
+7. [Acknowledgements](#acknowledgements)
+8. [Copyright](#copyright)
+9. [Footnotes](#footnotes)
+
+## Abstract
+
+Status exposes an Ethereum Provider object (`window.ethereum`) to JS code running inside the browser. It also provides bidirectional communication between JS code and Status.
+
+## Definitions
+
+| Term       | Description                                                                         |
+|------------|-------------------------------------------------------------------------------------|
+| **Webview**   | React Native wrapper for a platform-specific browser core implementation.                                        |
+| **Ethereum Provider** | A JS object (`window.ethereum`) injected into each web page opened in the browser providing web3 support. |
+| **Bridge** | A set of facilities allow bidirectional communication between JS code and Status. |
+
+## Implementation
+Status uses a [forked version](https://github.com/status-im/react-native-webview) of [react-native-webview](https://github.com/react-native-community/react-native-webview)  to display web or dapps content. The fork provides an Android implementation of JS injection before page load. It is required in order to properly inject Ethereum Provider object.
+
+Status injects two JS scripts: 
+  - [provider.js](https://github.com/status-im/status-react/blob/develop/resources/js/provider.js): `window.ethereum` object
+  - [webview.js](https://github.com/status-im/status-react/blob/develop/resources/js/webview.js): override for `history.pushState` used internally
+
+Dapps running inside a browser communicate with Status Ethereum node by means of a *bridge* provided by react-native-webview library. It allows for bidirection communication between browser and Status. In order to do so, it injects a special `ReactNativeWebview` object into each page it loads. 
+
+On Status (React Native) end, it provides `WebView.injectJavascript` function on a webview component that allows to execute arbitrary code inside the webview. Thus it is possible to inject a function call passing Status node response back to the Dapp.
+
+Below is the table briefly describing what functions/properties are used. More details available in package [docs](https://github.com/react-native-community/react-native-webview/blob/master/docs/Guide.md#communicating-between-js-and-native).
+
+| Direction | Side | Method |
+|-----------|------|-----------
+| Browser->Status | JS | `ReactNativeWebView.postMessage()`
+| Browser->Status | RN | `WebView.onMessage()`
+| Status->Browser | JS | `ReactNativeWebView.onMessage()`
+| Status->Browser | RN | `WebView.injectJavascript()`
+
+
+
+## Usage in Dapps
+
+`window.ethereum`
+
+### Properties
+
+#### `isStatus`
+Returns true.
```

Comment:
For me as a newbie to API it would be really helpful to read some examples on when different API methods applicable.
For example it wasn't clear for me how `isStatus` used until I googled that metamask also injects `windows.ethereum` object and Dapp can differentiate ethereum provider this way.

Reply:
Good point, will provide examples.